packages/gatsby-image/index.d.ts

```diff
@@ -20,13 +21,16 @@ interface FluidObject {
   tracedSVG?: string
   srcWebp?: string
   srcSetWebp?: string
+  media?: string
 }
 
 interface GatsbyImageProps {
   resolutions?: FixedObject
   sizes?: FluidObject
   fixed?: FixedObject
```

Comment:
how do we feel about moving to the following:
```
sizes?: FluidObject | FluidObject[]
fixed?: FixedObject | FixedObject[]
```

Reply:
Should `resolutions` and `sizes` support this? My understanding was they were legacy and discouraged.

Or were you suggesting to drop `fixedImages` & `fluidImages`?

Reply:
I talked to @KyleAMathews about this. I don't think either of us had an incredibly strong preference one way or another, but based on how this will be used (and the fact that it's already implemented as such), we felt good about keeping these as separate props.

**Usage example:**
```jsx
const sources = [
  {
    ...mobileImage.fluid,
    media: `(max-width: 767px)`,
  },
  {
    ...desktopImage.fluid,
    media: `(min-width: 768px)`,
  },
];

<Image
  alt={mobileImage.description}
  fluidImages={sources}
/>
```